arelle/plugin/validate/ESEF/resources/authority-validations.json

```diff
@@ -229,8 +229,7 @@
         "outdatedTaxonomyURLs": [
         ],
         "effectiveTaxonomyURLs": [
-            "https://www.esma.europa.eu/taxonomy/2021-03-24/esef_cor.xsd",
-            "http://www.esma.europa.eu/taxonomy/2021-03-24/esef_cor.xsd"
+            "https://xbrl.frc.org.uk/uksef/2022-01-01/uksef-2022-01-01_cor.xsd"
```

Comment:
No `http` version of this taxonomy?

Reply:
Guidance states that the filing must import the HTTPS entry point.

> _4.1 Guidance UKFRC22.1 – UKSEF taxonomy reference in issuer’s taxonomy extension_
Issuer taxonomy extensions targeting UKSEF 2022 MUST have a reference to the UKSEF
Core entry point (https://xbrl.frc.org.uk/uksef/2022-01-01/uksef-2022-01-01_cor.xsd).